src/sesclient.native/README.md

```diff
@@ -5,14 +5,17 @@ This command line executable demonstrates use of the [native-code software entit
 **This is draft documentation subject to change.**
 
 ## Working with sesclient.native
+
 This executable is intended to be used in conjunction with the [sestest command line utility](../sestest) to generate tokens.
 
 The executable expects the following parameters (in any order):
 
-| Parameter            | Required  | Definition                                                                                                                         |
-| -------------------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------- |
-| --url                | Mandatory | The URL at which the server should process requests. <br/> **Note**: must start with `https:`.                                      |
-| --thumbprint         | Mandatory | Thumbprint of a certificatein the server's SSL certificate chain. <br/> **Note**: cannot be the thumbprint of the root certificate. |
-| --common-name        | Mandatory | The common name of the certificate indicated by `--thumbprint`.                                                                    |
-| --token              | Mandatory | The token as returned by `sestest generate`.                                                                                       |
-| --application        | Mandatory | Unique identifier for the application being requested.                                                                             |
+|   Parameter   | Required  |                                                                   Definition                                                                    |
+| ------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------- |
+| --url         | Mandatory | The URL of the server that will process our request. <br/> **Note**: must start with `https:`.                                                  |
+| --token       | Mandatory | The token as returned by `sestest generate`.                                                                                                    |
+| --application | Mandatory | Unique identifier for the application being requested.                                                                                          |
+| --thumbprint  | Optional  | Thumbprint of an additional certificate to accept in the TLS certificate chain. <br/> **Note**: cannot be the thumbprint of a root certificate. |
+| --common-name | Optional  | The common name of the certificate indicated by `--thumbprint`.                                                                                 |
```

Comment:
What is the usage of `--common-name`?  Do I need to specify it if I specify `--thumbprint`?  If it's optional, when would I specify it and what does it do?

Reply:
It's a part of the authentication handshake used to ensure the server is genuine. The two parameters `--thumbprint` and `--common-name` have to be used together.